internal/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -42,6 +43,20 @@ const (
 		)
 		RETURNING id, token, type, args`
 
+	sqlDequeueByID = `
+		UPDATE jobs
+		SET token = $1, started_at = now()
+		WHERE id = (
+		  SELECT id
+		  FROM ready_jobs
+			  -- use ANY here, because "type in ()" doesn't work with bound parameters
+			  -- literal syntax for this is '{"a", "b"}': https://www.postgresql.org/docs/13/arrays.html
+		  WHERE id = ANY($2)
```

Comment:
`WHERE id = $2`, ANY not needed

Also need to make sure started_at is zero; I tihnk the easiest way to do this is to compare started_at to the default constructed go time and pass that as $3.

Reply:
Thanks! I fixed the where clause. `started_at` check is not needed, the `ready_jobs` view already has it.